ext/standard/proc_open.c

```diff
@@ -346,18 +342,26 @@ PHP_FUNCTION(proc_get_status)
 	}
 #endif
 
-	add_assoc_bool(return_value, "running", running);
+	add_assoc_bool(return_value, "running",  running);
 	add_assoc_bool(return_value, "signaled", signaled);
-	add_assoc_bool(return_value, "stopped", stopped);
+	add_assoc_bool(return_value, "stopped",  stopped);
 	add_assoc_long(return_value, "exitcode", exitcode);
-	add_assoc_long(return_value, "termsig", termsig);
-	add_assoc_long(return_value, "stopsig", stopsig);
+	add_assoc_long(return_value, "termsig",  termsig);
+	add_assoc_long(return_value, "stopsig",  stopsig);
```

Comment:
This alignment is not necessary.

Reply:
Do you mean you don't want things to be aligned (if it means adding whitespace)? Of course it's not necessary; I just did it because personally I find it easier to work on code that way.

If you don't like it, please just mention.